rust/src/records.rs

```diff
@@ -334,16 +330,6 @@ pub struct Channel {
     pub metadata: BTreeMap<String, String>,
 }
 
-pub fn system_time_to_nanos(d: &SystemTime) -> u64 {
-    let ns = d.duration_since(UNIX_EPOCH).unwrap().as_nanos();
-    assert!(ns <= u64::MAX as u128);
-    ns as u64
-}
-
-pub fn nanos_to_system_time(n: u64) -> SystemTime {
-    UNIX_EPOCH + Duration::from_nanos(n)
-}
```

Comment:
@james-rms can you recall if these were ever used for anything? Or are they helpers / part of the public API?

Reply:
There were never used for anything in any of our applications - I don't mind removing these.